### PR TITLE
Attrs doc

### PR DESCRIPTION
**Pull request by marimeireles**
Adding docs for custom attrs, noticed we didn't have these.
Only current way of knowing this is through exs. or looking at the code 👀

### CODE REVIEW

File: docs/changelog.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Documentation
 -------------
 
 - Fixed 'Direct usage of document is deprecated' warning in the getting started guide. ([#1052](https://github.com/pyscript/pyscript/pull/1052))
+- Adds docs for custom Pycsript attrs ([#1125](https://github.com/pyscript/pyscript/pull/1125))
```
Comment (FabioRosado) [inline docs/changelog.md:27]: small typo and suggestion

```suggestion
- Adds docs for custom Pyscript attributes (`py-*`) that allow you to add event listeners to an element  ([#1125](https://github.com/pyscript/pyscript/pull/1125))
```

File: docs/reference/API/attr_to_event.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# List of PyScript Attributes to Events:
+
+PyScript provides a convenient syntax for mapping JavaScript events to PyScript events, making it easy to connect events to HTML tags.
+
+For example, you can use the following code to connect the click event to a button:
+
+```
+<button id="my-button" py-onClick="foo()">Click me</button>
```
Comment (FabioRosado) [inline docs/reference/API/attr_to_event.md:8]: Maybeshould be `py-click` instead? I've been using this way in the tutorials and seems a bit confusing o have two different ways to handle click. What do you think?

File: docs/reference/API/attr_to_event.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+```
+
+Here is a list of all the available event mappings:
+
+| PyScript Event Name | DOM Event Name |
+|-------------------|----------------|
+| pys-onClick       | click          |
+| pys-onKeyDown     | keydown        |
```
Comment (FabioRosado) [inline docs/reference/API/attr_to_event.md:16]: These are deprecated, we shouldn't include it (they will be removed on the next release)

**Review by FabioRosado — CHANGES_REQUESTED** [review]
I have a WIP branch that adds the same to the API reference but I didn't include the table, I think we should merge this one in and then I'll build on top of this PR.

Currently, all these attributes are pretty impossible to figure out how to work with the event, we should also merge #1120 so users can easily access the event 😄

**Comment by marimeireles** [comment]
Ping! :)
Now should be good.
Thanks for the review Fabio.

File: docs/changelog.md [hunk 4]
```diff
@@ -24,6 +24,11 @@ Documentation
 -------------
 
 - Fixed 'Direct usage of document is deprecated' warning in the getting started guide. ([#1052](https://github.com/pyscript/pyscript/pull/1052))
+<<<<<<< Updated upstream
```
Comment (FabioRosado) [inline docs/changelog.md:27]: I think this was a bad merge 😄

Comment (marimeireles) [reply docs/changelog.md:27]: I used GH GUI 🙄 to make this commit
Never trust the GUIs

**Comment by marimeireles** [comment]
ping @FabioRosado is this still relevant or should we just close it?
Cheers :)

**Comment by FabioRosado** [comment]
I think is still relevant 😄 just wanted to check do we want to include the `ww` folder in this PR?

**Comment by marimeireles** [comment]
Okay, I fixed it, but now I'm waiting on Ted's feeback cause I've learned that there are some issues with these events that might be useful to add in here.
Once I figure that out I'll update the changelog and ping you again, 😅 sorry Fabio!

**Comment by FabioRosado** [comment]
Are the issues something that #1120 fixes maybe? 🤔

**Comment by FabioRosado** [comment]
Sounds good 👍

**Comment by marimeireles** [comment]
Hey @FabioRosado can you please add a ✅ ?
Before we forget this again 😅 and I have to rebase it again before we merge

**Review by FabioRosado — APPROVED** [review]

**Comment by FabioRosado** [comment]
Approved 👍  we probably should commit my suggestion about to fix the typo on PyScript 😄

**Comment by marimeireles** [comment]
K will do before merging! Thank you!! :)

**Pull request merged**